Cargo.toml

```diff
@@ -15,12 +15,12 @@
 [package]
 edition = "2018"
 name = "zerocopy"
-version = "0.7.21"
+version = "0.8.0-alpha"
 authors = ["Joshua Liebow-Feeser <joshlf@google.com>"]
 description = "Utilities for zero-copy parsing and serialization"
 license = "BSD-2-Clause OR Apache-2.0 OR MIT"
 repository = "https://github.com/google/zerocopy"
-rust-version = "1.60.0"
+rust-version = "1.56.0"
```

Comment:
We may need to update our testing infrastructure to support testing on more stable toolchains other than one MSRV and one stable. In particular, we'll want to test on every toolchain that enables some functionality in `build.rs`. If our MSRV is A, our `build.rs` emits a particular cfg for versions B and above (B > A), and our pinned stable is C (C > B), then if we only test on A and C in CI, it's possible for behavior which is gated behind the cfg to be broken on any version in the range [B, C). Thus, we should test on B as well. In the general case, we need the ability to test on the full list of toolchain versions that invoke distinct `build.rs` behavior.